<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629858083539' stop='1629858288452'>
    <name>Create new project </name>
    <title>Create new project </title>
    <test-cases>
        <test-case start='1629858083696' status='failed' stop='1629858288157'>
            <name>Should create new project successfully.</name>
            <title>Should create new project successfully.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='3ce64c68-035e-4fc4-8dd2-3c63326ed373-attachment.png' type='image/png' size='63933'/>
                <attachment title='Screenshot' source='137287ff-eae6-4abd-9327-fdc96918b545-attachment.png' type='image/png' size='63933'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (node:internal/timers:557:17)
                    at processTimers (node:internal/timers:500:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>